tests/conftest.py

```diff
@@ -1,29 +1,26 @@
 import os
 import shutil
-import pytest
 
-from kf_model_fhir import app
-from kf_model_fhir.config import DEFAULT_SITE_ROOT, DEFAULT_IG_CONTROL_FILE
+import pytest
 
 TEST_DIR = os.path.dirname(__file__)
 TEST_DATA_DIR = os.path.join(TEST_DIR, 'data')
 PROFILE_DIR = os.path.join(TEST_DATA_DIR, 'profiles')
 EXTENSION_DIR = os.path.join(TEST_DATA_DIR, 'extensions')
 EXAMPLE_DIR = os.path.join(TEST_DATA_DIR, 'examples')
+TEST_SITE_ROOT = os.path.join(TEST_DATA_DIR, 'site_root')
+TEST_IG_CONTROL_FILE = os.path.join(TEST_SITE_ROOT, 'ig.ini')
```

Comment:
This lets `site_root` and `ig.ini` sit under `tests/data` which makes them at the same level with `profiles`, `examples`, and `extensions`. For the `site_root` and `site_root/ig.ini` in the root directory, they are at the higher level than `profiles`, `examples`, and `extensions`. Is it intended?

Reply:
Yea the content in profiles, extensions, etc in the test data dir get copied over into test site_root which is actually just a template of an empty IG

Reply:
It’s so that we start w a fresh IG for each test. Some tests test valid conf resources and others test the invalid ones. Don’t want to mix and match invalid and valid resources